src/Collisions/babylon.collisionCoordinator.ts

```diff
@@ -393,7 +393,7 @@ module BABYLON {
             // Check all meshes
             for (var index = 0; index < this._scene.meshes.length; index++) {
                 var mesh = this._scene.meshes[index];
-                if (mesh.isEnabled() && mesh.checkCollisions && mesh.subMeshes && mesh !== excludedMesh &&  ((collisionMask & mesh.collisionMask) !== 0)) {
+                if (mesh.isEnabled() && mesh.checkCollisions && mesh.subMeshes && mesh !== excludedMesh &&  ((collisionMask & mesh.collisionGroup) === 0)) {

```

Comment:
i kept the invertion, everything collides by default, so it made more sense to me, that masks are used to ignore collision with other masks,

http://www.babylonjs-playground.com/#20PQBI#38